doc/tags.yml

```diff
@@ -287,27 +274,15 @@ mapping_topics:
   - nrf/drivers/wifi/nrf700x/nrf700x.html: ["wifi", "nrf70-series", "nrf7002", "nrf7001", "nrf7000"]
   - nrf/libraries/*.html: ["libraries", "kconfig"]
   - nrf/libraries/bin/bt_ll_acs_nrf53/index.html: ["nrf5340", "development-kits"]
-  - nrf/libraries/bin/lwm2m_carrier/index.html: ["nrf91-series", "nrf9161", "nrf9151", "nrf9160",
-                                                 "development-kits", "lte"]
   - nrf/libraries/bin/lwm2m_carrier/*.html: ["nrf91-series", "nrf9161", "nrf9151", "nrf9160",
                                              "development-kits", "lte"]
   - nrf/libraries/bin/lwm2m_carrier/certification.html: ["certification"]
-  - nrf/libraries/bluetooth_services/index.html: ["ble", "nrf52-series", "nrf52840", "nrf52833",
-                                                  "nrf52832", "nrf52820", "nrf52811", "nrf52810",
-                                                  "nrf52805", "development-kits"]
   - nrf/libraries/bluetooth_services/*.html: ["ble", "nrf52-series", "nrf52840", "nrf52833",
                                               "nrf52832", "nrf52820", "nrf52811", "nrf52810",
                                               "nrf52805", "development-kits"]
-  - nrf/libraries/bluetooth_services/mesh.html: ["ble", "blemesh", "nrf52-series", "nrf52840",
-                                                 "nrf52833", "nrf52832", "nrf52820", "nrf52811",
-                                                 "nrf52810", "nrf52805", "development-kits"]
-  - nrf/libraries/bluetooth_services/mesh/*.html: ["ble", "blemesh", "nrf52-series", "nrf52840",
-                                                   "nrf52833", "nrf52832", "nrf52820", "nrf52811",
-                                                   "nrf52810", "nrf52805", "development-kits"]
-  - nrf/libraries/gazell/index.html: ["gazell"]
+  - nrf/libraries/bluetooth_services/mesh.html: ["blemesh"]
```

Comment:
Is there a specific reason for removing other tags from the mesh folder?

Reply:
The reason to remove is, we already have these tags in "nrf/libraries/bluetooth_services/*.html". So, those will be applied.